specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json

```diff
@@ -0,0 +1,917 @@
+          },
+          {
+            "name": "publishedModelName",
+            "in": "path",
+            "description": "Specifies the name of the model to evaluate against.",
+            "required": true,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "name": "imageUrl",
+            "in": "body",
+            "description": "An {Iris.Web.Api.Models.ImageUrl} that contains the url of the image to be evaluated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ImageUrl"
+            }
+          },
+          {
+            "name": "application",
+            "in": "query",
+            "description": "Optional. Specifies the name of application using the endpoint.",
+            "required": false,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "$ref": "#/parameters/ApiKey"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImagePrediction"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CustomVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful ClassifyImageUrl request": {
+            "$ref": "./examples/ClassifyImageUrl.json"
+          }
+        }
+      }
+    },
+    "/{projectId}/classify/models/{publishedModelName}/image": {
+      "post": {
+        "tags": [
+          "ImagePredictionApi"
+        ],
+        "summary": "Classify an image and saves the result.",
+        "operationId": "ClassifyImage",
+        "consumes": [
+          "multipart/form-data",
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "projectId",
+            "in": "path",
+            "description": "The project id.",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-nullable": false
+          },
+          {
+            "name": "publishedModelName",
+            "in": "path",
+            "description": "Specifies the name of the model to evaluate against.",
+            "required": true,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "name": "application",
+            "in": "query",
+            "description": "Optional. Specifies the name of application using the endpoint.",
+            "required": false,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "name": "imageData",
+            "in": "formData",
+            "description": "Binary image data.",
```

Comment:
What formats are supported? Any specific dimensions/resolutions?

Reply:
Will udpate with a list supported formats and sizing restrictions (currently only size is a restriction).